src/Home.js

```diff
@@ -0,0 +1,351 @@
+            {/* Person Involved (if applicable) */}
+            {props.otherParty && 
+              <button className='userIcon calendarVariation' style={{backgroundColor: props.iconColor1}}>
+                {props.initials1}
+              </button>} 
+          </div>
+          {/* Event Name */}
+          <div className='eventName'>{props.name}</div>
+          {/* Event Location */}
+          {props.location}
+        </div>
+      </div>
+    )
+  }
+  // stores information for all events
+  const EVENTS = [
+    {
+      name: 'Class A',
+      start: new Date('2024-10-07T10:00'),
+      end: new Date('2024-10-07T13:00'),
+      location: 'Somewhere',
+      sideColor: getCSSVar('--purpleEvent'),
+      bgColor: getCSSVar('--purpleEventBg'),
+      otherParty: false
+    },
+    {
+      name: 'Class A',
+      start: new Date('2024-10-09T10:00'),
+      end: new Date('2024-10-09T13:00'),
+      location: 'Somewhere',
+      sideColor: getCSSVar('--purpleEvent'),
+      bgColor: getCSSVar('--purpleEventBg'),
+      otherParty: false
+    },
+    {
+      name: 'Class A',
+      start: new Date('2024-10-11T10:00'),
+      end: new Date('2024-10-11T13:00'),
+      location: 'Somewhere',
+      sideColor: getCSSVar('--purpleEvent'),
+      bgColor: getCSSVar('--purpleEventBg'),
+      otherParty: false
+    },
+    {
+      name: 'Class B',
+      start: new Date('2024-10-08T09:00'),
+      end: new Date('2024-10-08T11:00'),
+      location: 'Someplace',
+      sideColor: getCSSVar('--blueEvent'),
+      bgColor: getCSSVar('--blueEventBg'),
+      otherParty: false
+    },
+    {
+      name: 'Class B',
+      start: new Date('2024-10-10T09:00'),
+      end: new Date('2024-10-10T11:00'),
+      location: 'Someplace',
+      sideColor: getCSSVar('--blueEvent'),
+      bgColor: getCSSVar('--blueEventBg'),
+      otherParty: false
+    },
+    {
+      name: 'Lunch',
+      start: new Date('2024-10-08T13:30'),
+      end: new Date('2024-10-08T16:30'),
+      location: 'Elder Dining Commons',
+      sideColor: getCSSVar('--greenEvent'),
+      bgColor: getCSSVar('--greenEventBg'),
+      otherParty: true,
+        initials1: 'PO',
+        iconColor1: getCSSVar('--blueIcon')
+    }
+  ]
+  
+  
+
+  
+  
+  // displays home page
+  function Home() {
+    return(
+      <section className='homeBody'>
+        <section className='leftFrame'>
+          <div className='leftHead'>
+            <h2>Posts</h2>
+            <Filter/>
+            </div>
+            <PostList Posts={POSTS} profileList={false} userSavedList={false}/>
+            <CreateButton/>
+        </section>
+        <section className='rightFrame'>
+          <CalendarHead/>
+            <section className='calendarBodyContainer'>
+              <Times/>
+              <CalendarEvents/>
+            </section>
+        </section> 
+      </section>
+    )
+  }
+
+  export default Home;
+  export { Filter, CreateButton, POSTS, PostList };
```

Comment:
You're doing so much in one file! Highly recommend splitting this up into multiple files. If you still want things to be grouped, you can store these files in the same subdirectory of `src`, like in `src/home`. 
Generally, the scope of each file should be rather small.

Reply:
Lowkey some of the things ur exporting here dont exisst in this file, looks like they moved to variables.js